src/main/com/example011/radek/v1/Mines.java

```diff
@@ -0,0 +1,138 @@
+package main.com.example011.radek.v1;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Scanner;
+
```

Comment:
jeden empty line 

vytvor si enum StringConstant(at si vyzkousis, k cemu je enum) a hod do neho stringove constanty, ktere se ti tu opakujou, min. aspon tyto tri

TABULATOR("/t") 
emptyspace apod..
znak pro miny a pro vypis porovnavani pouzzij hodnoty z enumu ...

Reply:
Hotovo, snad je to takto správně